artemis-protocols/artemis-amqp-protocol/src/main/java/org/apache/activemq/artemis/protocol/amqp/converter/CoreAmqpConverter.java

```diff
@@ -514,7 +514,7 @@ private static Section convertBody(ServerJMSMessage message, Map<Symbol, Object>
             }
          } catch (Throwable ignored) {
```

Comment:
if you're really using the JIRa, rename the argument here as ``e`` or whatever other name you like, 
since it's not ignored any longer.

Reply:
Fair point! Fixed.